pytrap: failing tests on i386 arch

Building 32b package in Copr failes in all distro versions.

Sample output from a build:
```
copying build/lib.linux-i686-2.6/pytrap.so -> 
runTest (test.pytrapmodule_unittest.DeviceTest) ... ok
runTest (test.pytrapmodule_unittest.TrapCtxInitTest) ... ok
runTest (test.unirectemplate_unittest.DataAccessGetTest) ... ok
runTest (test.unirectemplate_unittest.DataAccessSetTest) ... FAIL
runTest (test.unirectemplate_unittest.DataTypesIPAddr) ... ok
runTest (test.unirectemplate_unittest.DataTypesTime) ... ok
runTest (test.unirectemplate_unittest.DeviceTest) ... ok
runTest (test.unirectemplate_unittest.Template2Test) ... ok
runTest (test.unirectemplate_unittest.Template3Test) ... ERROR
runTest (test.unirectemplate_unittest.TemplateSizeTest) ... ok
runTest (test.unirectemplate_unittest.TemplateTest) ... ok
======================================================================
ERROR: runTest (test.unirectemplate_unittest.Template3Test)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/nemea-pytrap-0.9.8/test/unirectemplate_unittest.py", line 291, in runTest
    self.assertTrue(a.U64 == 0x100000000)
OverflowError: long int too large to convert to int
======================================================================
FAIL: runTest (test.unirectemplate_unittest.DataAccessSetTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/builddir/build/BUILD/nemea-pytrap-0.9.8/test/unirectemplate_unittest.py", line 216, in runTest
    self.assertEqual(a.ABC, int(0))
AssertionError: 222 != 0
----------------------------------------------------------------------
Ran 11 tests in 0.013s
FAILED (failures=1, errors=1)
error: Bad exit status from /var/tmp/rpm-tmp.gByka6 (%check)
RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.gByka6 (%check)
Child return code was: 1
EXCEPTION: [Error()]
Traceback (most recent call last):
  File "/usr/lib/python3.5/site-packages/mockbuild/trace_decorator.py", line 88, in trace
    result = func(*args, **kw)
  File "/usr/lib/python3.5/site-packages/mockbuild/util.py", line 562, in do
    raise exception.Error("Command failed. See logs for output.\n # %s" % (command,), child.returncode)
mockbuild.exception.Error: Command failed. See logs for output.
 # bash --login -c /usr/bin/rpmbuild -bb --target i686 --nodeps /builddir/build/SPECS/nemea-pytrap.spec
```

Affected distro versions: `epel-6-i386`, `fedora-24-i386`, `fedora-25-i386`, `fedora-rawhide-i386`

--- a/pytrap/unirecmodule.c
+++ b/pytrap/unirecmodule.c
@@ -868,13 +868,13 @@ UnirecTemplate_set_local(pytrap_unirectemplate *self, char *data, int32_t field_
         }
         break;
     case UR_TYPE_UINT32:
-        longval = PyLong_AsLong(valueObj);
+        longval = PyLong_AsLongLong(valueObj);
         if (PyErr_Occurred() == NULL) {
             *((uint32_t *) value) = (uint32_t) longval;
         }
         break;
     case UR_TYPE_UINT64:
-        longval = PyLong_AsLong(valueObj);
+        longval = PyLong_AsLongLong(valueObj);
         if (PyErr_Occurred() == NULL) {
             *((uint64_t *) value) = (uint64_t) longval;
         }
@@ -892,7 +892,7 @@ UnirecTemplate_set_local(pytrap_unirectemplate *self, char *data, int32_t field_
         }
         break;
     case UR_TYPE_INT32:
-        longval = PyLong_AsLong(valueObj);
+        longval = PyLong_AsLongLong(valueObj);
         if (PyErr_Occurred() == NULL) {
             *((int32_t *) value) = (int32_t) longval;
         }

